code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -75,6 +75,11 @@
 	var/blockcount = 0		//number of times retried a blocked path
 	var/awaiting_beacon	= 0	// count of pticks awaiting a beacon response
 
+	var/turf/last_waypoint
+	var/bot_z_mode //SETTINGS: 10 = AI CALLED. 20 = PATROLLING. 30 = SOMEONE CALLED.
```

Comment:
Add DEFINES please

Reply:
I'm unsure of what you want me to do here
Do you want me to just define them like 
#define bot_z_mode ?

Reply:
#DEFINE BOT_Z_MODE_SETTINGS 10
etc

Reply:
Should go in `code/_DEFINES/` in one of the fitting files, not sure which one. If needed make a new one.